spinalcordtoolbox/straightening.py

```diff
@@ -163,7 +164,7 @@ def straighten(self):
 
         # Computation of the safe zone.
         # The safe zone is defined as the length of the spinal cord for which an axial segmentation will be complete
-        # The safe length (to remove) is computed using the safe radius (given as parameter) and the angle of the
+        # The safe length (to remove) is computed using the safe radius and the angle of the
```

Comment:
A lot of the computations below are moot and could be simplified, given that `radius_safe` is hard-coded to zero (and always has been, it seems?). But, let's not wake the dragons.